src/Sylius/Behat/Context/Ui/ShippingContext.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Ui;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Page\Shipping\ShippingMethodDetailsPageInterface;
```

Comment:
It should be:

```
Sylius\Behat\Page\Admin\ShippingMethod\ShowPageInterface
Sylius\Behat\Page\Admin\ShippingMethod\IndexPageInterface
```

WDYT? To be consistent with new pages, when someone migrates shipping method management to new UI, they will have less work. :)